transports/webtransport-websys/Cargo.toml

```
@@ -0,0 +1,48 @@
+[package]
+name = "libp2p-webtransport-websys"
+edition = "2021"
+rust-version = { workspace = true }
+description = "WebTransport for libp2p under WASM environment"
+version = "0.43.0"
```

```suggestion
version = "0.0.1"
```

Given that this is a new crate, why not start at `v0.1.0`?